src/mixins/triangle.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+/* eslint-disable default-case, consistent-return */
```

Comment:
Instead of disabling these, let's throw an error if a direction other than `top/bottom/left/right` is passed.

In the `switch` add a `default` statement that throws a helpful error so the user can fix their code!